source/administration/mobile-changelog.md

```diff
@@ -1,5 +1,39 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.36.0 Release
+- Release Date: October 16, 2020
+- Server Versions Supported: Server v5.25+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - **Upgrade to server version v5.25 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/administration/extended-support-release.html) (ESR) 5.19 has ended and upgrading to server ESR v5.25 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+
+### Improvements
+ - Added **Channel Settings > Notification Preferences** to add the ability to edit mobile push notification settings at the channel level.
+ - Server URL now autofills when opening the app from the mobile landing page.
```

Comment:
I can assume this is the landing page of the webapp, but it's not very clear imho